hecdssvue/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins{
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+
+    // The following plugin is used to build the OSGI bundle
+    id 'biz.aQute.bnd.builder' version '6.4.0'
+}
+
+jar {
+    archiveBaseName.set('hecdssvue')
+    archiveVersion.set('1.0.0.0')
+    archiveFileName.set("${archiveBaseName.get()}_${archiveVersion.get()}.jar")
+}
+
+configurations {
+    eclipseLibs
```

Comment:
I do not see how these eclipse libraries are used. They are not getting added to compile/runtime-time, and are only getting excluded from the copyDependencies

Reply:
Line 17 
> implementation.extendsFrom(eclipseLibs)
Is a common way to add the contents of one configuration to the other. Implementation is on the runtime and compile classpath. In this case, the eclipse jars have a dedicated configuration in order to make it easier to remove them when copying by using builtin gradle features (configuration - otherConfig = the set difference between the configurations)

Reply:
The eclipse libraries should not be runtime dependencies. They should be set to compileOnly.

Reply:
eclipse jars have been set to compileOnly. the copy task has also been removed from this module since it's only needed in the third-party module build.